libzstd/Cargo.toml

```diff
@@ -0,0 +1,33 @@
+[package]
+name = "scroll-zstd"
+version = "0.1.0"
+edition = "2021"
+
+[lib]
+crate-type = ["cdylib"]
+
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+aggregator = { git = "https://github.com/scroll-tech/zkevm-circuits.git", branch = "feat/rs_zstd_export" }
```

Comment:
is the encoder thread-safe (ask this because we only want to initialize an encoder globally once, not know if needs a lock), and would we use a stable branch later? currently, it's a feature branch.
@noel2004

Reply:
the encoder object itself is not thread-safe (and it should be not possible to handle input concurrencily).

The `da-compression` is not merged into stable branch yet so we can only use a feature branch now.

Reply:
got it.